widgets/use-assets/README.md

```diff
@@ -0,0 +1,8 @@
+# Use Assets
+This sample demonstrates how to use assets in a widget.
+
+## How to use the sample
+Clone the [sample repo](https://github.com/esri/arcgis-experience-builder-sdk-resources) and copy this widget's folder (within `samples/widgets`) to the `client/your-extensions/widgets` folder of your Experience Builder installation. 
+
+## How it works
+This widget requires an asset or use `props.context.folderUrl` to load an asset, then use it in an `<Icon>` component or a `<img>` tag.
```

Comment:
The `props` is the widget component prop, but the `context` is not react context, it's just a widget prop that is named context. So, I don't think we need to explain the props here because the very basic guide should have explained this one.

Reply:
I see, ok then. thanks!